src/core/hyper-ref/index.js

```diff
@@ -79,13 +79,15 @@ export default class HyperRef extends PureComponent<Props, State> {
       }
       return false;
     });
+    let time = 0;
     onEventBehaviors.forEach(behaviorElement => {
       const handler = this.createActionHandler(
         this.props.element,
         behaviorElement,
         this.props.onUpdate,
       );
-      handler();
+      setTimeout(handler, time);
```

Comment:
This change is a bit obscure, what purpose does it serve?

Reply:
This is an approach the codebase uses to make sure multiple triggering behaviors get applied properly to the DOM. For example with press handlers: https://github.com/Instawork/hyperview/blob/0cf2411492cfb9324b26a6180aa554b860cb8a8b/src/core/hyper-ref/index.js#L227

We need it because executing behavior A can modify the DOM resulting in a new top-level document node. Then, when the execute B, it's holding on to DOM elements that are no longer part of the top-level document, resulting in errors.

I think there's a better way to handle this by putting all behaviors in a queue in the core update code, rather than simulating a queue using timeouts. But for now, there's some precedent for this approach in the code.

Reply:
I guess the thing that I find confusing is the increment of `time` here - are we just deferring the execution to the next tick, in which case `setTimeout(handler, 0)` would work too, or does the extra millisecond waiting between each behavior has an incidence on the execution, and if so, what would happen if any given behavior took longer than 1ms to complete its DOM changes?

Reply:
We want each behavior to execute in a separate loop so that it operates on the latest DOM root, which is why we tick up the time for each behavior.

Reply:
I think using `setTimeout(handler, 0)` would have the same effect - each call to `setTimeout` would schedule some code to be run on the next loop, each of these blocking the loop until the next tick, causing subsequent timers to be pushed to the next event loop. To be verified / tested, I might be wrong but I think this is how it works.

Reply:
I think you're right, let me test it out.

Reply:
it worked, updated.